ratatui-widgets/src/chart.rs

```diff
@@ -328,6 +412,8 @@ pub struct Dataset<'a> {
     graph_type: GraphType,
     /// Style used to plot this dataset
     style: Style,
+    /// If using [`GraphType::Line`] you can specify different color ranges by value
+    multi_color_line_config: Option<MultiColorLine>,
```

Comment:
I think we can rename `multi_color_line_config` -> `multi_color_line` throughout this PR